logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultPathAssertions.kt

```diff
@@ -147,6 +150,17 @@ class DefaultPathAssertions : PathAssertions {
             .build()
     }
 
+    private inline fun <T : Path> pathTypeAssertion(
+        container: AssertionContainer<T>,
+        typeName: Translatable,
+        crossinline typeTest: (Path) -> Boolean
+    ) = changeSubjectToFileAttributes(container) { fileAttributesExpect ->
+        assertionBuilder.descriptive
+            .withTest(fileAttributesExpect) { it is Success && typeTest(it.path) }
+            .withPathAttributesFailureHint(fileAttributesExpect)
+            .withDescriptionAndRepresentation(DescriptionBasic.IS, typeName)
+            .build()
+    }
```

Comment:
Thinking about this again, I don't think it makes sense to have this for isRelative. A relative path is something like `test` or `test/bla.txt`.
Why do we want to read file attributes in this case? We don't need to do this to see if the path is relative or not. It is enough to call `isAbsolute()` on the path. personally I would skip all this. I think it could make sense for isAbsolute, although also there it is questionable if it really makes sense to check if the file exists.... nah... I would really just go and take the same approach as startsWith. I hope this is not contrary to what @jGleitz  told you

Reply:
Well, it's good point. And also the reason for failing builds. The implementation checks every time, is the path points to existing file. So it will fail every time, user tries to assert for _bla_ path

Reply:
@robstoll is absolutely right. I didn’t catch that when doing my review. Checking whether a path is absolute or relative needs no file system access at all. We don’t need to check file attributes, we don’t need `withAndWithoutSymlink` in the tests.

Reply:
fixed